script/release.py

```diff
@@ -101,6 +105,31 @@ def validate_release_env():
         exit("Please get twine via 'pip install gitchangelog' or 'pip install git+git://github.com/vaab/gitchangelog.git' for Python 3.7")
 
 
+def build():
+    shutil.rmtree(BUILT_APPIUM_DIR_PATH, ignore_errors=True)
+    os.system('{} setup.py install'.format(os.getenv('PYTHON_BIN_PATH')))
+
+
+def compare_file_count():
```

Comment:
consider using typing

Reply:
this function is expected to throw an exception in case of failure. Consider renaming it to `is_...` if it is supposed to return boolean